docs/topics/api/addons.rst

```diff
@@ -166,6 +166,7 @@ This endpoint allows you to fetch a specific add-on by id, slug or guid.
     :>json boolean has_privacy_policy: The add-on has a Privacy Policy (See :ref:`add-on EULA and privacy policy <addon-eula-policy>`).
     :>json string|object|null homepage: The add-on homepage (See :ref:`translated fields <api-overview-translations>`).
     :>json string icon_url: The URL to icon for the add-on (including a cachebusting query string).
+    :>json object icons: An object holding the URLs to an add-ons icon including a cachebusting query string. Currently exposes 32 and 64 pixels wide icons.
```

Comment:
`add-ons icon` should be `add-on icons` instead, I think. Also, not sure how to word this better but in it's current form it does not describe how the object is set up (i.e. it does not say the the sizes are the properties of the object, and the urls the values)

Reply:
> add-ons icon should be add-on icons instead, I think

Hmm, I wanted to name this so that it implies that the icon is actually the same and only the sizes change.

I'll think of something regarding the structure